cmd/svcat/binding/describe_cmd_test.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package binding
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/command"
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/test"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	svcatfake "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset/fake"
+	"github.com/kubernetes-incubator/service-catalog/pkg/svcat"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	k8sfake "k8s.io/client-go/kubernetes/fake"
+
+	_ "github.com/kubernetes-incubator/service-catalog/internal/test"
+)
+
+func TestDescribeCommand(t *testing.T) {
+	const namespace = "default"
+	testcases := []struct {
+		name         string
+		fakeBindings []string
+		bindingName  string
+		expected     string
+		wantError    bool
+	}{
+		{
+			name:         "describe non existing binding",
+			fakeBindings: []string{},
+			bindingName:  "mybinding",
+			expected:     "unable to get binding '" + namespace + ".mybinding': servicebindings.servicecatalog.k8s.io \"mybinding\" not found",
+			wantError:    true,
+		},
+		{
+			name:         "describe existing binding",
+			fakeBindings: []string{"mybinding"},
+			bindingName:  "mybinding",
+			expected:     "  Name:        mybinding  \n  Namespace:   " + namespace + "    \n  Status:                 \n  Secret:                 \n  Instance:               \n\nParameters:\n  No parameters defined\n",
```

Comment:
For successful cases, I suggest not looking for an expected string and just relying on `wantError: false` since the full cli output is adequately tested elsewhere.

Reply:
Cool, in that case I changed "expected" for "expectedError" and moved the check for only when wantError is true.